src/pages/guidelines/typography/code.mdx

```diff
@@ -103,124 +103,34 @@ You can find a full reference of the type styles that are available on the
 You can include a token in your Sass file by writing:
 
 ```scss
-@import '@carbon/type/scss/styles';
+@use '@carbon/type';
 
 @include type-style('productive-heading-01');
 ```
 
-In addition, if the type style you are using has a fluid style then you can pass
-in `true` as the second argument to `type-style` to enable fluid styles. For
-example:
-
-```scss
-@import '@carbon/type/scss/styles';
-
-@include type-style('token-name', true);
-```
-
-### Font-face
-
-`@carbon/type` supports three font-face definitions that you can use to add IBM
-Plex to your project. These font-face definitions include support for:
-
-- IBM Plex Mono
-- IBM Plex Sans
-- IBM Plex Serif
-
-For most projects, only IBM Plex Mono and IBM Plex Sans is necessary. We also
-provide IBM Plex Serif if you are building an editorial or marketing project.
-
-These font-face definitions are pulling the above fonts from Google Fonts. As a
-result, they are not intended to be used as a production asset for your project.
-While you can depend on these for bootstrapping your project, we highly
-recommend using the fonts from the `@ibm/plex` package and hosting them on a
-global CDN.
-
-You can include each font-face definition by including the corresponding file
-and calling its mixin. For example, if you wanted to include IBM Plex Mono in
-your project you would write the following in your Sass file:
-
-```scss
-@import '@carbon/type/scss/font-face/mono';
-
-@include font-face-mono();
-```
-
-Similarly, you can include IBM Plex Sans and IBM Plex Serif by writing:
-
-```scss
-@import '@carbon/type/scss/font-face/mono';
-@import '@carbon/type/scss/font-face/sans';
-@import '@carbon/type/scss/font-face/serif';
-
-@include font-face-mono();
-@include font-face-sans();
-@include font-face-serif();
-```
-
 ### Type classes
 
-The recommended way to style your application will be to use our
-[type styles](#type-styles). However, we also offer helper CSS classes for
-specific use-cases. These are also helpful when quickly prototyping a project.
-
-You can include type classes in your project by writing the following in your
-Sass file:
-
-```scss
-@import '@carbon/type/scss/classes';
```

Comment:
Could you add a code snippet that details how to bring in the classes into a product's CSS?